src/connectors/aws_collect.py

```diff
@@ -662,6 +662,27 @@
         ('created_at', 'TIMESTAMP_NTZ'),
         ('updated_at', 'TIMESTAMP_NTZ'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-snapshots.html
+    'ec2_describe_snapshots': [
+        ('recorded_at', 'TIMESTAMP_LTZ'),
+        ('account_id', 'STRING'),
+        ('region', 'STRING'),
+        ('error', 'VARIANT'),
+        ('description', 'STRING'),
```

Comment:
the first data column should be `data_encryption_key_id`, to store the `DataEncryptionKeyId` key in the response from AWS